image-processing/main.py

```diff
@@ -5,10 +5,48 @@
 # Then runs the provided R-script to generate processed images
 
 import os
+import cv2
+import requests
+import numpy as np
+
 from google.cloud import vision
 from google.cloud import storage
 
 
+def face_check(face):
```

Comment:
Maybe this should all be in another file specific to cloud vision then you could move all these constants to be global constants. This will make our code much better and modularized too.

Reply:
sorry I was not clear enough here but I like what you did with the constants, they look clean! :D

I was actually thinking of having another file like `face_detection.py` containing all these cloud vision logic (including its specific constants) then you could `from face_dection import face_check` from `main.py` or something like that. It's not a big deal now though since our main is not huge yet but might be worth refactoring if you feel like it. Otherwise it already looks clean as is.

Reply:
Tbh even I was thinking the same. But let’s wait and see if there is a need to that in future :)